c_binding/include/libstoragemgmt/libstoragemgmt.h

```diff
@@ -1044,6 +1045,172 @@ int LSM_DLL_EXPORT lsm_system_read_cache_pct_update(lsm_connect *c,
                                                     uint32_t read_pct,
                                                     lsm_flag flags);
 
+/**
+ * Get a list of batteries for this storage system.
+ * The supper capacitor will also be included.
+ * New in version 1.3
+ * @param [in]      conn            Valid connection @see
+ *                                  lsm_connect_password
+ * @param[in]       search_key      Search key (NULL for all)
+ * @param[in]       search_value    Search value
+ * @param [out]     batteries       An array of lsm_battery types
+ * @param [out]     count           Number of batteries
+ * @param [in]      flags           Reserved set to zero
+ * @return LSM_ERR_OK on success else error reason
```

Comment:
I think the function documentation could benefit from stating that information explicitly rather than leaving the user confused (as I was) and making them find the correct information in the codebase.